### PR TITLE
get vars from file and replace vars recurrently 

### PR DESCRIPTION
**Pull request by snigle**

### CODE REVIEW

File: cli/venom/run/cmd.go [hunk 1]
```diff
@@ -38,6 +44,7 @@ var (
 
 func init() {
 	Cmd.Flags().StringSliceVarP(&variables, "var", "", []string{""}, "--var cds='cds -f config.json' --var cds2='cds -f config.json'")
+	Cmd.Flags().StringVarP(&varFile, "var-from-file", "", "", "--var-from-file filename.yaml : yaml|json, must contains map[string]string'")
```
Comment (yesnault) [inline cli/venom/run/cmd.go]: Can you add an example of usage in Readme too?

File: executors/ovhapi/executor.go [hunk 2]
```diff
@@ -28,11 +29,12 @@ type Headers map[string]string
...
-	NoAuth   bool   `json:"no_auth" yaml:"noAuth"`
-	Path     string `json:"path" yaml:"path"`
-	Body     string `json:"body" yaml:"body"`
-	BodyFile string `json:"bodyfile" yaml:"bodyfile"`
+	Method      string            `json:"method" yaml:"method"`
+	NoAuth      bool              `json:"no_auth" yaml:"noAuth"`
+	Path        string            `json:"path" yaml:"path"`
+	Body        string            `json:"body" yaml:"body"`
+	BodyFile    string            `json:"bodyfile" yaml:"bodyfile"`
+	QueryParams map[string]string `json:"queryparams,omitempty" yaml:"queryparams,omitempty"`
```
Comment (MarcAudefroy) [inline executors/ovhapi/executor.go]: QueryParams url.Values?

**Comment by MarcAudefroy** [comment]
Don't merge please. Need works.

File: templater.go [hunk 3]
```diff
@@ -12,9 +12,10 @@ type Templater struct {
 	Values map[string]string
 }
 
-func newTemplater(values map[string]string) *Templater {
-	if values == nil {
-		values = make(map[string]string)
+func newTemplater(inputValues map[string]string) *Templater {
```
Comment (fsamin) [inline templater.go]: i don't understand this modification. Could you explain ?

Comment (yesnault) [inline templater.go]: Yep, seen, with parallel > 1

**Pull request merged**